CHANGES.md

```diff
@@ -6,6 +6,10 @@
 
 - Added `ScreenOverlay` support to `KMLDataSource`. [#9864](https://github.com/CesiumGS/cesium/pull/9864)
 
+##### Fixes :wrench:
+
+- fix the bug of ktx2 texture minfilter.
```

Comment:
Please also merge `main` in again and put this under a new `##### Fixes :wrench:` section for `### 1.88 - 2021-12-1`. Thanks!

```suggestion
- Fixed a bug with .ktx2 textures having an incorrect minification filter. [#9876](https://github.com/CesiumGS/cesium/pull/9876/)
```